### PR TITLE
refactor(site): Refactor workspace schedule page

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
- Refactor workspace schedule page
- Refactor workspace settings page

Before:
Screen Shot 2023-04-10 at 15 03 20

After:
Screen Shot 2023-04-10 at 15 03 05

### CODE REVIEW

File: site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx [hunk 1]
```diff
@@ -48,15 +50,15 @@ export const Language = {
...
+  daySundayLabel: "Sun",
+  dayMondayLabel: "Mon",
+  dayTuesdayLabel: "Tue",
+  dayWednesdayLabel: "Wed",
+  dayThursdayLabel: "Thu",
+  dayFridayLabel: "Fri",
+  daySaturdayLabel: "Sat",
   startTimeLabel: "Start time",
-  startTimeHelperText: "Your workspace will automatically start at this time.",
+  startTimeHelperText: "Time the workspace will automatically start.",
```
Comment (Kira-Pilot) [inline site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:61]: I think the old string is a little more grammatical - is there a reason we changed it?

File: site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx [hunk 2]
```diff
@@ -280,24 +282,24 @@ export const WorkspaceScheduleForm: FC<
...
-                  onChange={handleToggleAutostart}
-                  color="primary"
-                />
-              }
-              label={Language.startSwitch}
-            />
+    <HorizontalForm onSubmit={form.handleSubmit}>
+      <FormSection
+        title="Autostart"
+        description="Select the time and days of week you want your workspace starting automatically."
```
Comment (Kira-Pilot) [inline site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:288]:
```suggestion
        description="Select the time and days of week on which you want your workspace starting automatically."
```

File: site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx (hunk 1 above)
Comment (Kira-Pilot) [reply site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:61]: Maybe we don't need this helper text at all anymore considering we have a label and also a general field description.

Comment (BrunoQuaresma) [reply site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:61]: That is true, what do you think is best?

Comment (Kira-Pilot) [reply site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx:61]: I would just take it out!

**Review by Kira-Pilot — APPROVED** [review]
Big improvement!

**Pull request merged**